README.md

```diff
@@ -27,7 +27,39 @@ Notebooks uses AI Platform version 1.14, we had to code in Python 3.5 (no
 f-strings or type hinting)
 
 ### Jupyter via remote execution into AI Platform Notebook Instance
-Coming Soon - Some documentation already complete by team
+Prerequisites: 
+1. You must have Anaconda/Jupyter Notebook installed 
+2. If using a windows machine, you must have PuTTY installed
```

Comment:
Is there anything different if using a mac? I know we only have windows here, but if we have info to make this more useful to others that stumble on the repo, that would be great!

Reply:
Yes- mac's have a built in ssh client so PuTTY is not needed - I will update!

Reply:
Oh yeah! It's been so long since I've used a Mac!